SMLHelper/Initializer.cs

```diff
@@ -46,6 +47,7 @@ internal static void InitializeOld()
 
         internal static void Initialize()
         {
+            CustomFishPatcher.Patch(harmony);
```

Comment:
Is there any risk in patching the custom fish _before_ patching the `TechTypes`?

Reply:
All this does is harmony patch Creature.Start() to spawn the fish